src/components/Popup/Popup.styl

```diff
@@ -0,0 +1,33 @@
+.backdrop {
+	position: absolute;
+	top: 0;
+	left: 0;
+	bottom: 0;
+	right: 0;
+	text-align: center;
+	
+	&:before {
+		content: '';
+		display: inline-block;
+		vertical-align: middle;
+		width: 0;
+		height: 100%;
+	}
+		
+	&:not(&_isModal) {
+		pointer-events: none;
+		.container {
+			pointer-events: auto;
+		}
+	}
+		
+	&_isModal.backdrop_closeOnClickAway {
+		cursor: pointer;
+	}
+}
+	
+.container {
```

Comment:
Why should `window` be used when we have a convension about naming all root elements `containers` in components?